### PR TITLE
Update First PR Welcome/Merge Message in config.yml

### LINKED ISSUE #805: Update First PR Welcome/Merge Message
Hi, this is a [first-timers-only issue](https://code.publiclab.org/#r=all). This means we've worked to make it more legible to folks who either **haven't contributed to our codebase before, or even folks who haven't contributed to open source before**. 

If that's you, we're interested in helping you take the first step and can answer questions and help you out as you do. Note that we're especially interested in contributions from people from groups underrepresented in free and open source software!

We know that the process of creating a pull request is the biggest barrier for new contributors. This issue is for you 💝

If you have contributed before, **consider leaving this one for someone new**, and looking through our general [help wanted](https://github.com/publiclab/plots2/labels/help-wanted) issues. Thanks!

### 🤔 What you will need to know.

Nothing. This issue is meant to welcome you to Open Source :) We are happy to walk you through the process.

### 📋 Step by Step

- [ ] 🙋 **Claim this issue**: Comment below. If someone else has claimed it, ask if they've opened a pull request already and if they're stuck -- maybe you can help them solve a problem or move it along!

- [ ] 📝 **Update** the file [.github/config.yml](https://github.com/publiclab/PublicLab.Editor/blob/main/.github/config.yml) in the `PublicLab.Editor` repository (press the little pen Icon) and edit the line as shown below. 

[See this page](https://code.publiclab.org/#r=all) for some help in taking your first steps!

Below is a "diff" showing in red (and a `-`) which lines to remove, and in green (and a `+`) which lines to add:

```diff
@@ -13,18 +13,21 @@ newIssueWelcomeComment: |
 # Comment to be posted to on PRs from first time contributors in your repository
 newPRWelcomeComment: |
  Thanks for opening this pull request!  This space is [protected by our Code of Conduct](https://publiclab.org/conduct) - and we're here to help.
- Please make sure you've bumped up the version number in the repository's [package.json file](https://github.com/publiclab/PublicLab.Editor/blob/master/package.json) and that you've incorporated your changes in the [/dist/ sub-directory](https://github.com/publiclab/PublicLab.Editor/tree/master/dist).
+ Please make sure you've bumped up the version number in the repository's [package.json file](https://github.com/publiclab/PublicLab.Editor/blob/main/package.json) and that you've incorporated your changes in the [/dist/ sub-directory](https://github.com/publiclab/PublicLab.Editor/tree/main/dist) by running `grunt build`.
  `Dangerbot` will test out your code and reply in a bit with some pointers and requests.
- Also please refer [here](https://github.com/publiclab/PublicLab.Editor#installation) for installation help 💿    
- There may be some errors, **but don't worry!** We'll work through them with you! 👍🎉😄    
- It would be great if you can tell us your Twitter handle so we can thank you properly?
+ Also please refer [here](https://github.com/publiclab/PublicLab.Editor#installation) for installation help 💿
+ There may be some errors, **but don't worry!** We'll work through them with you! 👍🎉😄   
+ <br />
+ One thing that can help to get started is to make sure you've included a link back to the original issue you're solving, in the format `fixes #0000` (for example). And to make sure the PR title describes what you're trying to do! (often it can be the same as the issue title) Thanks! 🙌 
+ <br /> 
+ Then, you can [say hello in our chatroom](https://gitter.im/publiclab/publiclab) & share a link to this PR to get a review! 👋 ✅ 
 # Configuration for first-pr-merge - https://github.com/behaviorbot/first-pr-merge
 
 # Comment to be posted to on pull requests merged by a first time user
 firstPRMergeComment: |
  Congrats on merging your first pull request! 🙌🎉⚡️ 
- Your code will likely be published to PublicLab.org in the next few days. In the meantime, can you tell us your Twitter handle so we can thank you properly?
- Do join our [weekly check-in](https://github.com/publiclab/plots2/issues/5627) to share your this week goal and the awesome work you did 😃.
+ Your code will likely be published to PublicLab.org in the next few days.
+ Do join our [weekly check-in](https://github.com/publiclab/plots2/issues/9600) to share your this week goal and the awesome work you did 😃.
  Reach out to someone else working on theirs on [Public Lab's code welcome page](https://code.publiclab.org#r=all). Thanks!
  Now that you've completed this, you can help someone else take their first step!
  See: [Public Lab's coding community!](https://code.publiclab.org)
@@ -37,4 +40,5 @@ firstPRMergeComment: |
  <p>People often get stuck at the same steps, so you might be able to help someone get unstuck, or help lead them to some documentation that'd help. Reach out and be encouraging and friendly! 😄 🎉 </p>
  <p>Read about <a href="https://github.com/publiclab/plots2/issues/3458">how to help support another newcomer here</a>, or find <a href="https://github.com/publiclab/plots2/labels/support">other ways to offer mutual support here</a>.</p>
  </details>
+ <p>Join our Gitter channel <a href="https://gitter.im/publiclab/publiclab">here</a> to connect with other contributors</p> :inbox_tray:
 # It is recommended to include as many gifs and emojis as possible
```

- [ ] 💾 **Commit** your changes

- [ ] 🔀 **Start a Pull Request**. There are two ways how you can start a pull request:

1. If you are familiar with the terminal or would like to learn it, [here is a great tutorial](https://egghead.io/series/how-to-contribute-to-an-open-source-project-on-github) on how to send a pull request using the terminal.

2. You can also [edit files directly in your browser](https://help.github.com/articles/editing-files-in-your-repository/) and open a pull request from there. 

- [ ] 🏁 **Done** Ask in comments for a review :)

### Please keep us updated

💬⏰ - We encourage contributors to be respectful to the community and provide an update **within a week** of claiming a first-timers-only issue. We're happy to keep it assigned to you as long as you need if you update us with a request for more time or help, but if we don't see any activity a week after you claim it we may reassign it to give someone else a chance. Thank you in advance! 

If this happens to you, don't sweat it! Grab another open issue.

### Is someone else already working on this?

🔗- We encourage contributors to link to the original issue in their pull request so all users can easily see if someone's already started on it. 

👥- **If someone seems stuck, offer them some help!** Otherwise, [take a look at some other issues you can help with](https://code.publiclab.org/#r=all). Thanks!

### 🤔❓ Questions?

Leave a comment below!

### PR DESCRIPTION
**Pull request by taylorkaiulani**
Fixes #805. Replaced lines in newPRWelcomeComment and firstPRMergeComment and added line 42 to details element.

### CODE REVIEW

File: .github/config.yml [hunk 1]
```diff
@@ -13,18 +13,21 @@ newIssueWelcomeComment: |
...
+ Then, you can [say hello in our chatroom](https://gitter.im/publiclab/publiclab) & share a link to this PR to get a review! 👋 ✅
 # Configuration for first-pr-merge - https://github.com/behaviorbot/first-pr-merge
 
 # Comment to be posted to on pull requests merged by a first time user
 firstPRMergeComment: |
  Congrats on merging your first pull request! 🙌🎉⚡️ 
- Your code will likely be published to PublicLab.org in the next few days. In the meantime, can you tell us your Twitter handle so we can thank you properly?
- Do join our [weekly check-in](https://github.com/publiclab/plots2/issues/5627) to share your this week goal and the awesome work you did 😃.
+ Your code will likely be published to PublicLab.org in the next few days.
+ Do join our [weekly check-in](https://github.com/publiclab/plots2/issues/9600) to share your this week goal and the awesome work you did 😃.
```
Comment (TildaDares) [inline .github/config.yml:30]: Hi @taylorkaiulani, I know this isn't in the original issue but would you mind changing `share your this week goal` to `share your goal for this week`. Thanks and sorry for the inconvenience!

Comment (taylorkaiulani) [reply .github/config.yml:30]: Will do!

Comment (taylorkaiulani) [reply .github/config.yml]: Okay, I committed the change. Let me know if there's anything else I need to do!

**Review by TildaDares — APPROVED** [review]
Thanks @taylorkaiulani. Great job!

**Comment by TildaDares** [comment]
Thanks @taylorkaiulani!

**Pull request merged**